qfieldsync/gui/synchronize_dialog.py

```diff
@@ -65,7 +65,10 @@ def __init__(self, iface, offline_editing, parent=None):
         self.button_box.button(QDialogButtonBox.Save).clicked.connect(
             lambda: self.start_synchronization()
         )
-        import_dir = self.preferences.value("importDirectoryProject")
+        try:
+            import_dir = self.preferences.value("importDirectoryProject")
+        except ValueError:
+            import_dir = None
         if not import_dir:
             import_dir = self.preferences.value("importDirectory")
```

Comment:
Does this line also need to be hardened?

Reply:
@m-kuhn , no, that setting isn't a dictionary.